code/go/0chain.net/blobbercore/challenge/challenge.go

```diff
@@ -432,8 +340,68 @@ func commitChallenge(id string) {
 		zap.String("load", elapsedLoad.String()),
 		zap.String("commit_on_chain", elapsedCommitOnChain.String()),
 		zap.String("commit_on_db", elapsedCommitOnDb.String()),
+		zap.String("time_taken", time.Since(startTime).String()),
 	)
 
-	go cMap.Delete(c.ChallengeID) //nolint
+	//go cMap.Delete(c.ChallengeID) //nolint
+
+}
+
+func loadTodoChallenges(doProcessed bool) {
+	db := datastore.GetStore().GetDB()
+	now := time.Now().Unix()
+	from := now - int64(config.Configuration.ChallengeCompletionTime.Seconds())
+
+	db = db.Model(&ChallengeEntity{}).
+		Where("created_at > ? AND status in (?)", from, Accepted)
+
+	if doProcessed {
+		db.Model(&ChallengeEntity{}).
```

Comment:
What does this do?
It does seem to affect anything

Reply:
`true` is designed for blobber restarts, and `false` is designed for timer.

Reply:
have a look at `startWorkers`

Reply:
Actually my point was, Unless you assign it like `db=db.Model()`, it does nothing

Reply:
no this is how it works @lpoli 
Please refer to documentation